bookings/bookings.go

```diff
@@ -0,0 +1,44 @@
+// Wrapper for Zoho Bookings API from https://www.zohoapis.com/bookings/v1/json/
+
+package bookings
+
+import (
+	zoho "github.com/schmorrison/Zoho"
+	"math/rand"
+)
+
+const (
+	BookingsAPIEndpoint string = "https://www.zohoapis.in/bookings/v1/json/"
+	GetAppointmentModule string = "getappointment"
+	FetchWorkspacesModule string = "workspaces"
+	FetchServicesModule string = "services"
+	FetchStaffModule string = "staffs"
+	FetchResourceModule string = "resources"
+	BookAppointmentModule string = "appointment"
+	RescheduleAppointmentModule string = "rescheduleappointment"
+	UpdateAppointmentModule string = "updateappointment"
```

Comment:
I am not sure if we are using `strings` or have defined a type like `zoho.Module`. Check `crm` subpackage for reference.

Reply:
I have checked the `crm` subpackage, in `crm.go`, `Module` is used which is a type alias for `string`